locales/tr-TR/translation.json

```diff
@@ -0,0 +1,280 @@
+{
+  "By default, the address is generated in a random group. You can select the group you want the address to be generated in using the Advanced options.": "Varsayılan olarak adres rastgele grup içinte üretildi. Gelişmiş seçenekleri kullanarak adresin üretilmesini istediğiniz grubu seçebilirsiniz.",
+  "Use an existing wallet": "Varolan bir cüzdanı kullan",
+  "Please choose whether you want to create a new wallet or import an existing one.": "Lütfen yeni bir cüzdan oluşturmak mı ya da varolan cüzdanı içe aktarmak mı istediğinizi seçin.",
+  "New wallet": "Yeni cüzdan",
+  "Import wallet": "Cüzdanı içe aktar",
+  "Amount must be greater than {{ minAmountInAlph }}": "Miktar {{ minAmountInAlph }} 'dan daha fazla olmalıdır",
+  "Could not connect to the {{ currentNetwork }} network.": "{{ currentNetwork }} ağına bağlantı yapılamadı.",
+  "Note that if activated, \"{{ address }}\" will not be the default address anymore.": "Eğer etkinleştirilirse, \"{{ address }}\" adresinin artık varsayılan olmayacağını unutmayınız.",
+  "Error while sweeping address {{ from }}": "{{ from }} 'dan sürükleme adresinde hata oluştu",
+  "Remove wallet \"{{ walletName }}\"": " \"{{ walletName }}\" cüzdanını kaldır",
+  "Enter password for \"{{ switchToWalletName }}\"": " \"{{ switchToWalletName }}\" için şifre girin",
+  "{{ amount }} words entered": "{{ amount }} kelime girildi",
+  "Address label": "Adres etiketi",
+  "The wallet is offline.": "Cüzdan çevirimdışı.",
+  "Discreet mode": "Gizli mod",
+  "Default address": "Varsayılan adres",
+  "Settings": "Ayarlar",
+  "Copied to clipboard!": "Panoya kopyalandı!",
+  "Copy to clipboard": "Panoya kopyala",
+  "Copied": "Kopyalandı",
+  "Official Wallet": "Resmi Cüzdan",
+  "The easiest way to get started with Alephium.": "Alephium ile başlamanın en kolay yolu.",
+  "Genesis TX": "Genesis TX(Blokzinciri yaratılış işlemi)",
```

Comment:
I can see why you would want to add the translation in parenthesis here, but unfortunately, it creates a pretty long badge... How can we improve this?

<img width="1415" alt="image" src="https://user-images.githubusercontent.com/1579899/218668617-25756d0e-095e-4730-8396-5be3ce1770f0.png">